Common/UnitDefinitions/RotationalStiffness.json

```diff
@@ -22,8 +22,8 @@
       ]
     },
     {
-      "SingularName": "PoundForceFootPerDegrees",
-      "PluralName": "PoundForceFeetPerDegrees",
+      "SingularName": "PoundForceFootPerDegree",
```

Comment:
Great initiative, however this would be a breaking change and not something we can do.
I propose reverting these and adding a comment in #563 to address it there.